test/distributed/org/apache/cassandra/distributed/test/accord/AccordDropTableTest.java

```diff
@@ -35,8 +34,7 @@ public void dropTable() throws IOException
         int steps = 5;
         try (Cluster cluster = Cluster.build(3)
                                       .withoutVNodes()
-                                      .withConfig(c -> c.with(Feature.values())
```

Comment:
please revert, `Feature.values()` is 100% intentional

Reply:
Looks like I forgot to push the rebased version, where this was gone.